Hymns/Common/SongResultViewModel.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+import SwiftUI
+
+class SongResultViewModel<DestinationView>: ObservableObject, Identifiable where DestinationView: View {
```

Comment:
I see it is an ObservableObject but what takes advantage of that? I don't see any @published variables or any places that has SongResultViewModel as an ObservedObject? Will this come into play later?

Reply:
you're right. It doesn't need to be an ObservableObject